frontend/src/contexts/group.tsx

```diff
@@ -67,9 +69,10 @@ const GroupProvider: FC = () => {
     id: currentGroup?.id || -1,
     activeAgreements: activeAgreements?.core_agreements || [],
     archivedAgreements: archivedAgreements?.core_agreements || [],
-    categories: categoriesData?.core_categories || [],
+    categories: categoriesData?.core_categories || [{ name: 'New Category', id: NaN }],
```

Comment:
New category shouldn't be included in group's categories

Reply:
Also, the string "New Category" should be dynamic, implemented in `bank.ts` and `en.ts` (see https://github.com/consenz-io/webapp/blob/main/frontend/README.md#31-strings)

Reply:
both fixed